resources/web/docs_js/utils.js

```diff
@@ -5,9 +5,12 @@ export function get_base_url(href) {
              .replace(/^http:/, 'https:');
 }
 
+const VERSION_REGEX = /[^\/]+\/+([^\/]+\.html)/;
```

Comment:
because I like to verify regexes: this is matching on exactly one directory + one file, e.g. `foo/bar.html`, but not `foo/bar/baz.html` or `foo.html`